app/src/androidTest/java/org/simple/clinic/bloodsugar/sync/BloodSugarSyncAndroidTest.kt

```diff
@@ -0,0 +1,79 @@
+package org.simple.clinic.bloodsugar.sync
+
+import androidx.test.runner.AndroidJUnit4
+import com.f2prateek.rx.preferences2.Preference
+import io.reactivex.Single
+import org.junit.Before
+import org.junit.Rule
+import org.junit.rules.RuleChain
+import org.junit.runner.RunWith
+import org.simple.clinic.TestClinicApp
+import org.simple.clinic.TestData
+import org.simple.clinic.bloodsugar.BloodSugarMeasurement
+import org.simple.clinic.bloodsugar.BloodSugarRepository
+import org.simple.clinic.patient.SyncStatus
+import org.simple.clinic.rules.ServerAuthenticationRule
+import org.simple.clinic.sync.BaseSyncCoordinatorAndroidTest
+import org.simple.clinic.sync.BatchSize
+import org.simple.clinic.sync.SyncConfig
+import org.simple.clinic.sync.SyncGroup
+import org.simple.clinic.sync.SyncInterval
+import org.simple.clinic.util.Optional
+import org.simple.clinic.util.RxErrorsRule
+import javax.inject.Inject
+import javax.inject.Named
+
+@RunWith(AndroidJUnit4::class)
+class BloodSugarSyncAndroidTest : BaseSyncCoordinatorAndroidTest<BloodSugarMeasurement, BloodSugarMeasurementPayload>() {
+
+  @Inject
+  lateinit var repository: BloodSugarRepository
+
+  @Inject
+  @field:Named("last_blood_sugar_pull_token")
+  lateinit var lastPullToken: Preference<Optional<String>>
+
+  @Inject
+  lateinit var syncApi: BloodSugarSyncApi
+
+  @Inject
+  lateinit var bpSync: BloodSugarSync
```

Comment:
I think this was a copy-paste error, the field is still named `bpSync`.

Reply:
Done